src/views/home/components/list/list.view.tsx

```diff
@@ -28,31 +27,9 @@ interface ListProps {
   onClose: () => void;
 }
 
-const filterChainList = (chainList: ChainList, value: string): ChainList => ({
-  ...chainList,
-  items: chainList.items.filter((chain) => chain.name.toUpperCase().includes(value.toUpperCase())),
-});
-
-const filterTokenList = (tokenList: TokenList, value: string): TokenList => ({
-  ...tokenList,
-  items: tokenList.items.filter(
-    (token) =>
-      token.name.toUpperCase().includes(value.toUpperCase()) ||
-      token.symbol.toUpperCase().includes(value.toUpperCase())
-  ),
-});
-
-const List: FC<ListProps> = ({ placeholder, list, onClose }) => {
+const List: FC<ListProps> = ({ list, onClose }) => {
   const classes = useListStyles();
-  const [values, setValues] = useState(list);
-
-  const onChange = (event: React.ChangeEvent<HTMLInputElement>) => {
-    const value = event.target.value.trim();
-    const filtered: List =
-      list.type === "chain" ? filterChainList(list, value) : filterTokenList(list, value);
-
-    setValues(filtered);
-  };
+  const [values] = useState(list);
```

Comment:
If we no longer modify this state we can remove it and use the `list` straight.

Reply:
Done!